src/presenters/nImagePresenter.js

```diff
@@ -162,4 +162,10 @@ class ImagePresenter {
 	}
 }
 
-module.exports = ImagePresenter;
+module.exports = (options) => {
+	console.log({ options });
+	if (options.data) {
+		const image = new ImagePresenter(options.hash);
+		return options.fn({ image });
```

Comment:
these need supported until all react n-image use is gone:
https://github.com/Financial-Times/n-image/blob/master/src/components/image.js#L2
https://github.com/Financial-Times/n-image/blob/master/src/components/image.js#L8
but will break with these changes.

what do you think of either 1) leave image.js as is and have a second file nImagePresenter that exports this (requires the same image.js file) OR 2) export different things depending on whether you have options.fn or not